.gitignore

```diff
@@ -9,3 +9,4 @@ dist/
 *.build.js.map
 *.build.spec.js
 *.build.spec.js.map
+.DS_Store
```

Comment:
as an FYI you can add this to your global .gitignore then you'll never need to add it to projects again